src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -537,15 +554,11 @@ private static (Project project, ImmutableArray<Document> documents) GetProjectA
             return (project, documents);
         }
 
-        // make sure we always move remote workspace forward
-        private int _solutionVersion = 0;
```

Comment:
this was an ugly test hack.  I've removed it and updated the tests themselves to do the right thing.